docs/ref/utils.txt

```diff
@@ -476,11 +476,14 @@ https://web.archive.org/web/20110718035220/http://diveintomark.org/archives/2004
 
         from django.utils.functional import cached_property
 
-        @cached_property
```

Comment:
It might be easier to understand by showing only the changes rather than repeating everything.
Maybe something like:
```
    Here, ``friends()`` will be called twice. Since the instance ``person`` in
    the view and the template are the same, decorating the ``friends()`` method
    with ``@cached_property`` can avoid that::
    
        from django.utils.functional import cached_property
        
        class Person(models.Model):
        
            @cached_property
            def friends(self):
                ...
```